internal/pkg/template/env.go

```diff
@@ -123,10 +123,11 @@ type EnvOpts struct {
 
 // HTTPConfig represents configuration for a Load Balancer.
 type HTTPConfig struct {
```

Comment:
Why do we need these changes 🤔 I don't see any CloudFormation template modifications am I misunderstanding something?

Reply:
Removed these changes.